### DISCUSSION TITLE
Background color of option in select multiple

### DISCUSSION THREAD

**Discussion by danielmoessner** [tailwindlabs/tailwindcss#6027] [Help]
How can I change the background color when another option is selected? It should be blue as well.

[https://play.tailwindcss.com/Xr3o9RzXEs](https://play.tailwindcss.com/Xr3o9RzXEs)

**Comment by NovusOpus** [comment]
Well it's working fine.
To select both at the same time drag your mouse over both while holding click, or Hold down the Ctrl (windows) or Command (Mac) button then click the options you want, to select multiple options. 

That's the way it works. if you just single click one, then the other, it's going to act as a single select.

Also to see your defined `checked:bg-blue-500` background, after finishing selecting your options click outside of the select box or it's going to stay in the browser default `option:checked` (blue in chrome) background color.

For all of this unless you really have to implement this way, its advised to use checkboxes:

> Definition and Usage:
> 
> The multiple attribute is a boolean attribute.
> 
> When present, it specifies that multiple options can be selected at once.
> 
> Selecting multiple options vary in different operating systems and browsers:
> 
> For windows: Hold down the control (ctrl) button to select multiple options
> For Mac: Hold down the command button to select multiple options
> 
> Because of the different ways of doing this, and because you have to inform the user that multiple selection is available, it is more user-friendly to use checkboxes instead.
[W3schools](https://www.w3schools.com/tags/att_select_multiple.asp)
